nextstep-mvc/src/main/java/nextstep/mvc/tobe/adapter/ControllerAdapter.java

```diff
@@ -0,0 +1,20 @@
+package nextstep.mvc.tobe.adapter;
+
+import nextstep.mvc.asis.Controller;
+import nextstep.mvc.tobe.ModelAndView;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class ControllerAdapter implements HandlerAdapter {
+    @Override
+    public boolean isHandle(Object handler) {
+        return Controller.class.isAssignableFrom(handler.getClass());
```

Comment:
```suggestion
        return handler instanceof Controller;
```
instanceof를 쓰면 좀더 간결해질 것 같네요

Reply:
감사합니다!